ls1intum/Orion
# `Feature` Local Testing for Students

### Motivation and Context
Students should be able to execute tests locally. Therefore a button that allows students to set up tests with a few clicks is a good way to simplify the process.

### Description
- supports Java and Kotlin
- supports Maven and Gradle Projects

### Steps for Testing
Prerequisites: We need a Java Artemis exercise that allows local test execution and the deadline has to be over.
The student user can now clone the test repository with the respective button in Artemis and Orion should set the tests up.
This works for Java Maven and Gradle Exercises.

1. Set up the exercise using Orion
2. Set up test repository
3. verify that the run configuration is correct

### Review Progress


#### Code Review
#### Manual Tests

### Screenshots

There is no real UI! The Change creates a new Folder and sets up a run configuration
![grafik](https://github.com/ls1intum/Orion/assets/25010075/53af0e76-096e-4499-8507-5be6c1390173)
